### PR TITLE
FileScan upload methods support

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileScanClient.java [hunk 1]
```diff
@@ -96,13 +112,45 @@ public FileScanClient build() {
...
+
+	public AcceptedResponse requestUploadURL(FileScanUploadURLRequest request)
+		throws PangeaException, PangeaAPIException {
+		TransferMethod tm = request.getTransferMethod();
+		if (tm == null) {
+			tm = TransferMethod.PUT_URL;
+		}
+
+		if (tm == TransferMethod.MULTIPART) {
+			throw new PangeaException(String.format("%s not supported. User scan() instead", tm), null);
```
Comment (ggallien13) [inline packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileScanClient.java:143]: Looks like an extra `r`. Should be `Use scan() instead`.

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileUploader.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	}
+
+	public void uploadFile(String url, TransferMethod transferMethod, FileData fileData)
+		throws PresignedURLException, PangeaException {
+		if (transferMethod == null) {
+			throw new PangeaException("transferMethods should not be null", null);
+		}
+
+		if (transferMethod == TransferMethod.MULTIPART) {
+			throw new PangeaException(String.format("%s not supported. User scan() instead", transferMethod), null);
```
Comment (ggallien13) [inline packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileUploader.java:33]: Should be `Use` instead of `User`.

**Review by ggallien13 — APPROVED** [review]

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileUploader.java (hunk 2 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileUploader.java]: Fixed

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileScanClient.java (hunk 1 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/file_scan/FileScanClient.java:143]: Fixed

**Review by ggallien13 — APPROVED** [review]

**Pull request merged**